casts/cards/src/UIMediaCard/UIMediaCard.tsx

```diff
@@ -0,0 +1,62 @@
+import * as React from 'react';
+import { StyleSheet } from 'react-native';
+import LinearGradient from 'react-native-linear-gradient';
+import { ColorVariants, makeStyles, Theme, useTheme } from '@tonlabs/uikit.themes';
+import { TouchableOpacity } from '@tonlabs/uikit.controls';
+import { UILayoutConstant, UISkeleton } from '@tonlabs/uikit.layout';
+import type { UIMediaCardProps } from './types';
+import { Title } from './Title';
+import { QuickView } from './QuickView';
+import { UIConstant } from '../constants';
+
+export function UIMediaCard({
+    contentType,
+    title,
+    onPress,
+    source,
+    loading,
+    testID,
+}: UIMediaCardProps) {
+    const theme = useTheme();
+    const styles = useStyles(theme);
+    return (
+        <UISkeleton show={!!loading} style={styles.skeleton}>
+            <TouchableOpacity testID={testID} onPress={onPress} style={styles.container}>
+                <QuickView style={styles.quickView} source={source} contentType={contentType} />
+                <LinearGradient
+                    start={{ x: 0, y: 1 }}
+                    end={{ x: 0, y: 0 }}
+                    colors={[UIConstant.linearGradientStart, UIConstant.linearGradientEnd]}
+                    style={styles.gradient}
+                />
+                <Title title={title} />
+            </TouchableOpacity>
+        </UISkeleton>
+    );
+}
+
+const useStyles = makeStyles((theme: Theme) => ({
+    skeleton: {
+        /**
+         * The radius of the skeleton has to be made a little smaller
+         * so that the corners of the content are not visible
+         */
+        borderRadius: UIConstant.uiMediaCard.borderRadius - 2,
+    },
+    container: {
+        borderRadius: UIConstant.uiMediaCard.borderRadius,
+        backgroundColor: theme[ColorVariants.BackgroundOverlay],
+        padding: UILayoutConstant.contentOffset,
+        justifyContent: 'flex-end',
+        aspectRatio: 1,
+        overflow: 'hidden',
+    },
+    quickView: {
+        ...StyleSheet.absoluteFillObject,
+        zIndex: -20,
```

Comment:
Oh, I believe that the problem happens due to the fact that you try to wrap `2` components inside one `TouchableOpacity`, which is an anti-pattern.  This was mentioned somewhere in RN docs, that you can wrap only a single component with the Touchables...

Reply:
I did a single component and applied `zIndex` only for web